tests/rados/rados_mon_tests.py

```diff
@@ -0,0 +1,99 @@
+import logging
```

Comment:
it is a good practice to have a module level description providing the purpose of this module.